app/src/main/java/fr/free/nrw/commons/explore/images/SearchImageFragment.java

```diff
@@ -71,7 +72,10 @@ private void saveQuery(String query) {
         // Newly searched query...
         if (recentSearch == null) {
             recentSearch = new RecentSearch(null, query, new Date());
+        }else {
```

Comment:
Trivial but usually we write
```
}
else {
```
I think.

Reply:
Ok, @nicolas-raoul  I have updated it and will take care of it from next time.